app/locales/sk/notification.json

```diff
@@ -0,0 +1,8 @@
+{
+    "notification_main_text": "Skontroluj svoj prienik denných dát",
+    "notification_title": "Analyzuj svoj prienik",
+    "notification_data_not_available": "\n\nNemáš žiadne stiahnuté trasy!\n\n",
+    "notification_warning_text": "Stlačením nasledujúceho tlačidla môžete vygenerovať náhodné križovatky.\nVarovanie: Tieto simulované križovatky nie sú skutočné",
```

Comment:
I dont get what this is trying to say

Reply:
Perhaps "prieniky" (translates as intersection) instead of "križovatky" (translates as traffic lights)